protocol-units/bridge/integration-tests/src/lib.rs

```diff
@@ -31,11 +31,13 @@ use ethereum_bridge::{
 use movement_bridge::{Config as MovementConfig};
 use std::sync::{Arc, RwLock};
 
+#[cfg(test)]
```

Comment:
You don't need to feature flag anything inside the `/integration-tests` lib as this who binary will be conditionally built. 

I think we have to use a custom feature we could call it bridge-tests
in bridge-integration-tests Cargo.toml you do
```
[package]
name = "bridge-integration-tests"
version = "0.1.0"
[features]
bridge-tests = []
```
then in the root workspace Cargo.toml  you do
```
[features]
bridge-tests = ["bridge-integration-tests"]
```
and
```
[dev-dependencies]
bridge-integration-tests = { path = "./path/to/package", features = ["bridge-tests"] }
```

Something like this, It's a way to create your own testing binaries that get conditionally built. Without having to conform to the rather limiting workspace `/tests/` rule.